subsys/bluetooth/host/Kconfig

```diff
@@ -284,8 +284,15 @@ config BT_SIGNING
 	  This option enables data signing which is used for transferring
 	  authenticated data in an unencrypted connection.
 
+config BT_SMP_SC_PAIR_ONLY
+	bool "Disable legacy pairing"
+	help
+	  This option disables LE legacy pairing and forces LE secure connection
+	  pairing.
```

Comment:
Please add a comment as to which security levels can be achieved with this option enabled

Reply:
Added. All security levels can be used with this. Level 1 because you start with that, and Level 2 because unauthenticated pairing is still allowed.